src/main/java/io/reactivex/Maybe.java

```diff
@@ -1238,7 +1238,7 @@
     }
 
     /**
-     * Returns a Flowable that emits one item after a specified delay on a specified Scheduler.
+     * Returns a Maybe that emits one item after a specified delay on a specified Scheduler.
```

Comment:
This PR was created on the current 2.x so after you posted that earlier PR in any case if you cleaned this one up already I could also just revert this one change